src/tcgwars/logic/impl/gen8/AmazingVoltTackle.groovy

```diff
@@ -496,13 +496,34 @@ public enum AmazingVoltTackle implements LogicCardInfo {
         weakness L
         bwAbility "Torrent Awakening", {
           text "If this Pokémon has a Memory Capsule card attached to it, each player's Pokémon has no Abilities."
-          actionA {
+          onActivate {
+            bg.em().run(new CheckAbilities())
+          }
+          onDeactivate {
+            bg.em().run(new CheckAbilities())
+          }
+          getterA IS_ABILITY_BLOCKED, {holder->
+            if (self.cards.any { it.name == "Memory Capsule" } && thisAbility.name != holder.effect.ability.name && holder.effect.ability instanceof BwAbility) {
+              holder.object = true
+            }
+          }
+          getterA IS_GLOBAL_ABILITY_BLOCKED, {holder->
+            if (self.cards.any { it.name == "Memory Capsule" } && thisAbility.name != holder.effect.ability.name && holder.effect.ability instanceof BwAbility) {
+              holder.object = true
+            }
+          }
```

Comment:
I think "has no abilities" needs to be `GET_ABILITIES`

Reply:
Okay, I'll give that a try. I pretty much copied this from Garbodor BKP since it was pretty similar. Would this need changed for that as well?

Reply:
Not sure how recennt that is but I checked this onne from sun moonn and it seems fairly implemented 

```      case ALOLAN_MUK_58:
        return evolution (this, from:"Alolan Grimer", hp:HP120, type:PSYCHIC, retreatCost:4) {
          weakness PSYCHIC
          bwAbility "Power of Alchemy", {
            text "Each Basic Pokémon in play, in each player's hand, and in each player's discard pile has no Abilities."
            getterA (GET_ABILITIES, BEFORE_LAST) {h->
              if (h.effect.target.basic && h.effect.target != self) {
                h.object.keySet().removeIf{it instanceof BwAbility}
              }
            }
            getterA IS_GLOBAL_ABILITY_BLOCKED, {Holder h->
              if ((h.effect.target as Card).cardTypes.is(BASIC)) {
                h.object=true
              }
            }
            onActivate {
              new CheckAbilities().run(bg)
            }
            onDeactivate {
              new CheckAbilities().run(bg)
            }
          }
```